expression/builtin_time.go

```diff
@@ -1785,8 +1785,91 @@ type builtinAddTimeSig struct {
 
 // eval evals a builtinAddTimeSig.
 // See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_addtime
+// ADDTIME(expr1,expr2)
+// only support expr1 is datetime expression ,not time
 func (b *builtinAddTimeSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("ADDTIME")
+	args, err := b.evalArgs(row)
+	//sc := b.ctx.GetSessionVars().StmtCtx
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	if args[0].IsNull() || args[1].IsNull() {
+		return d, nil
+	}
+	s, err := args[1].ToString()
+	arg1, err := types.ParseDuration(s, getFsp(s))
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	switch tp := args[0].Kind(); tp {
+	case types.KindMysqlTime:
+		arg0 := args[0].GetMysqlTime()
+		tmpDuration := arg0.Add(arg1)
+		result, err := tmpDuration.ConvertToTime(arg0.Type)
```

Comment:
what's the result of sql:
`select addtime("2017-01-01 01:01:01.11", "01:01:01.11111");`
will it be 

>  2017-01-01 02:02:02.***22*** 


In MySQL, we get:
mysql> select addtime("2017-01-01 01:01:01.11", "01:01:01.11111");
+-----------------------------------------------------+
| addtime("2017-01-01 01:01:01.11", "01:01:01.11111") |
+-----------------------------------------------------+
>| 2017-01-01 02:02:02.***221110***                          |

+-----------------------------------------------------+
1 row in set (0.00 sec)